crates/pop-parachains/src/new_parachain.rs

```diff
@@ -75,12 +80,72 @@ pub fn instantiate_standard_template(
 	Ok(tag)
 }
 
+/// Create a new tanssi compatible container chain.
+///
+/// # Arguments
+///
+/// * `template` - template to generate the parachain from.
```

Comment:
```suggestion
/// * `template` - template to generate the container-chain from.
```

Container-chain seems to be the preferred styling, at least from their readme